extensions/big-data-cluster/package.json

```diff
@@ -80,6 +88,16 @@
 					"command": "bigDataClusters.command.mount",
 					"when": "nodeType=~/^mssqlCluster/ && nodeType!=mssqlCluster:message && nodeSubType=~/^(?!:mount).*$/",
 					"group": "1mssqlCluster@10"
+				},
+				{
+					"command": "bigDataClusters.command.refreshmount",
+					"when": "nodeType == mssqlCluster:folder && nodeSubType=~/^:mount:$/",
```

Comment:
Any reason you're using a regex here but not for delete?

Reply:
My mistake - I was doing A/B testing to see which one worked (both do). Will fix